sdk/Cargo.toml

```diff
@@ -68,6 +70,7 @@ stronghold = [ "iota_stronghold" ]
 events = [ ]
 ledger_nano = [ "iota-ledger-nano" ]
 participation = [ "storage" ]
+message_interface = [ "backtrace", "fern-logger" ]
```

Comment:
Why `"fern-logger"` here?

Reply:
It's only used for the `init_logger` function of the message interface

Reply:
Ah Brord added the function also to main crate, I thought it was only in the bindings